modules/govuk/lib/puppet/parser/functions/data_sync_times.rb

```diff
@@ -0,0 +1,38 @@
+module Puppet::Parser::Functions
+  newfunction(:data_sync_times, :type => :rvalue, :doc => <<-EOS
+Return information about the data sync times, depending on the argument passed
+    EOS
+  ) do |args|
+    start_hour = 22
+    start_minute = 0
+    finish_hour = 8
+    finish_minute = 0
+
+    if args.size != 1
+      raise ArgumentError, "data_sync_times: Wrong number of arguments " +
+      "(given #{args.size})"
+    end
```

Comment:
If we specify the number of given arguments, it would probably be useful to also suggest the expected number. Something like:
```suggestion
	finish_minute = 0
	expected_args = 1

    if args.size != expected_args
      raise ArgumentError, "data_sync_times: Wrong number of arguments " +
      "(given #{args.size}, expected #{expected_args})"
    end
```

(with or without parentheses, I don't mind them with `raise`)